includes/class-coblocks-block-assets.php

```diff
@@ -122,11 +122,87 @@ public function editor_assets() {
 				),
 				'cropSettingsOriginalImageNonce' => wp_create_nonce( 'cropSettingsOriginalImageNonce' ),
 				'cropSettingsNonce'              => wp_create_nonce( 'cropSettingsNonce' ),
+				'customIcons'                    => $this->get_custom_icons(),
 			)
 		);
 
 	}
 
+	/**
+	 * Load custom icons from the theme directory, if they exist
+	 *
+	 * @return array Custom icons array if they exist, else empty array.
+	 */
+	public function get_custom_icons() {
+
+		$config = array();
+		$icons  = glob( get_stylesheet_directory() . '/coblocks/icons/*.svg' );
```

Comment:
Does this work with child themes? Should this be `get_theme_file_uri` or `get_theme_file_path`? instead?

Reply:
This does work with child themes. Specifically the custom icons must be uploaded to the child theme if you child theme is active.